db/seeds.rb

```diff
@@ -1,9 +1,68 @@
-# This file should ensure the existence of records required to run the application in every environment (production,
-# development, test). The code here should be idempotent so that it can be executed at any point in every environment.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Example:
-#
-#   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
-#     MovieGenre.find_or_create_by!(name: genre_name)
-#   end
+user1 = User.where(email: "test1@example.com").first_or_create(password: "password", password_confirmation: "password")
+user2 = User.where(email: "test2@example.com").first_or_create(password: "password", password_confirmation: "password")
+
+apartments = [
+  {
+    street: '129 West 81st Street',
+    unit: '5A',
+    city: 'NY',
+    state: 'NY',
+    square_footage: '1,000',
+    price: '2000',
+    bedrooms: 1,
+    bathrooms: 1.0,
+    pets: 'no pets allowed',
+    image: 'https://www.hollywoodreporter.com/wp-content/uploads/2015/06/seinfeld_apt.jpg',
+    user_id: user1
```

Comment:
The foreign key gets created down on line 65 when an apartment gets created on an existing user.